x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension.tsx

```diff
@@ -5,45 +5,535 @@
  * 2.0.
  */
 
-import React, { memo, useEffect } from 'react';
-import { EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import React, { memo, useState, useCallback, useEffect } from 'react';
+import { EuiForm, EuiFlexGroup, EuiFlexItem, EuiCheckbox, EuiRadio, EuiSelect } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
 import type { PackagePolicyCreateExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import styled from 'styled-components';
+import { useLicense } from '../../../../../common/hooks/use_license';
 
 /**
  * Exports Endpoint-specific package policy instructions
  * for use in the Ingest app create / edit package policy
  */
+
+const ENDPOINT_INTEGRATION_CONFIG_KEY = 'ENDPOINT_INTEGRATION_CONFIG';
+
+const NGAV = 'NGAV';
+const EDR_ESSENTIAL = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.endpointDropdownOptionEDREssential',
+  {
+    defaultMessage: 'EDR Essential',
+  }
+);
+const EDR_COMPLETE = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.endpointDropdownOptionEDRComplete',
+  {
+    defaultMessage: 'EDR Complete',
+  }
+);
+
+const ENDPOINT = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.endpointDropdownOption',
+  {
+    defaultMessage: 'Endpoint',
+  }
+);
+const CLOUD_SECURITY = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.cloudSecurityDropdownOption',
+  {
+    defaultMessage: 'Cloud Security',
+  }
+);
+const INTERACTIVE_ONLY = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.cloudEventFiltersInteractiveOnly',
+  {
+    defaultMessage: 'Interactive only',
+  }
+);
+const ALL_EVENTS = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.cloudEventFiltersAllEvents',
+  {
+    defaultMessage: 'All events',
+  }
+);
+const PREVENT_MALWARE = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.cloudEventFiltersPreventionMalware',
+  {
+    defaultMessage: 'Prevent Malware',
+  }
+);
+const PREVENT_MALICIOUS_BEHAVIOUR = i18n.translate(
+  'xpack.securitySolution.createPackagePolicy.stepConfigure.cloudEventFiltersPreventionMaliciousBehaviour',
+  {
+    defaultMessage: 'Prevent Malicious Behaviour',
+  }
+);
+
+const SelectConfigurationSettingsText = styled.div`
```

Comment:
Could you use Eui's theme variables for all of the css properties in this file?

It is preferable that we don't hard code these value and instead use EUI theme variables so that we don't deviate from the standard ui library. For any value that you don't see in the UI them variables, you should use one that is closes to the value you need. Example: below I see `fond-size: 12.25` - you probably should be using `theme.eui.euiFontSizeXS`.  It would also be best to (if possible) create styled components from Eui components and only override the properties that you need customization to


I would suggest that you use EUI components before actually creating custom styled components. Example: below you have one called `SubduedText` - can Eui's `<EuiText color="subdued"`>` be used?